### DISCUSSION TITLE
Building drvs in repl should be possible

### DISCUSSION THREAD

**Discussion by 573** [nix-community/nix-on-droid#226] [Q&A] [answered]
Taking as an example https://github.com/Gerschtli/nix-config/blob/c1cf8ec859d510a58799cc3be097392d3edd520d/flake.nix one can via

```
nix repl
> :lf .#
> drv = outputs.nixosConfigurations.neon.pkgs.vlc
> :b drv
```

etc. inspect the properties of any derivation in the pkgs set to be installed on machine `neon`.

I wonder if and how the same or similar can be done for `outputs.nixOnDroidConfigurations.oneplus5` somehow but I did not find a way yet.

**Comment by Gerschtli** [comment, marked as answer]
nixos and home-manager export their `pkgs` already so I think it would be a nice addition. Basically adding `inherit pkgs;` in https://github.com/t184256/nix-on-droid/blob/master/modules/default.nix#L36.

Feel free to create a PR, not sure when I can do it.

Would be then accessible via `outputs.nixOnDroidConfigurations.oneplus5.pkgs`.

BTW: I am glad, my nix-config seems to be helpful :)
